test/parallel/test-repl-top-level-await.js

```diff
@@ -174,7 +174,7 @@ async function ordinaryTests() {
       'undefined',
     ]],
     // Regression test for https://github.com/nodejs/node/issues/43777.
-    ['await Promise.resolve(123), Promise.resolve(456)', 'Promise {', { line: 0 }],
+    ['await Promise.resolve(123), Promise.resolve(456)', 'Promise { 456 }', { line: 0 }],
```

Comment:
What changed?

Reply:
I'm not *quite* sure, but I'm looking into it. Drafting until resolved.

Reply:
Originally, the `domain` module added some symbol properties to track? the promise. Now that the `domain` module is removed, these properties no longer are added.

You can see this via:
```mjs
const domain = require('domain');
console.log(Reflect.ownKeys(Promise.resolve()));
domain.create();
console.log(Reflect.ownKeys(Promise.resolve()));
```
```
[]
[ Symbol(async_id_symbol), Symbol(trigger_async_id_symbol) ]
```